bin/wc-latest-version.php

```diff
@@ -27,11 +27,10 @@ function filter_valid_versions() {
  * @return 3 minor versions, in the order of oldest to newest.
  */
 function get_last_three_minor($versions) {
-	$count = 0;
 	$last_minor_version = '';
 	$results = [];
 	foreach ($versions as $index => $version_number) {
-		$current_minor_version = substr($version_number, 0, strrpos($version_number, '.') + 1);
+		$current_minor_version = substr($version_number, 0, strrpos($version_number, '.'));
```

Comment:
@frosso I removed the `$count` in this last commit. As well, I noticed I had a `+1` on the `substr`. If `$version_number` is `4.12.1`, then the `substr` would had changed it to `4.12.`. I removed the `+1` so that it is `4.12`. In terms of functionality, it doesn't make any differences, just better for my eyes. 

I also added a "How to test the script" step in the description.

Reply:
Oh cool, thanks! Good either ways :D